scripts/cat/history.py

```diff
@@ -139,6 +143,8 @@ def make_dict(cat):
             "possible_scar": cat.history.possible_scar,
             "scar_events": cat.history.scar_events,
             "murder": cat.history.murder,
+            "possible_exile": cat.history.possible_exile,
```

Comment:
What is the use of "possible_exile"? Are there events that can cause time-delayed exile?